### PR TITLE
Problem: latest v0.44 SDK is not used

### PR DESCRIPTION
**Pull request by tomtau**
Solution: bumped SDK + upgraded Go to 1.17

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -26,6 +26,103 @@ require (
 
 )
 
+require (
```
Comment (tomtau) [inline go.mod:29]: this is generated when I ran `go mod tidy` https://golang.org/doc/go1.17

Comment (tomtau) [reply go.mod:29]: > For the go command to correctly resolve transitive imports using the pruned module graph, the go.mod file for each module needs to include more detail about the transitive dependencies relevant to that module. If a module specifies go 1.17 or higher in its go.mod file, its go.mod file now contains an explicit require directive for every module that provides a transitively-imported package. (In previous versions, the go.mod file typically only included explicit requirements for directly-imported packages.)

**Comment by tomtau** [comment]
for the dependency stuff: https://github.com/crypto-org-chain/chain-main/issues/618

**Review by yihuang — APPROVED** [review]
LGTM

**Pull request merged**